src/unix/solarish/x86_64.rs

```diff
@@ -0,0 +1,129 @@
+pub type greg_t = ::c_long;
+
+s! {
+    pub struct __c_anonymous_fpchip_state {
+        pub cw: u16,
+        pub sw: u16,
+        pub fctw: u8,
+        pub __fx_rsvd: u8,
+        pub fop: u16,
+        pub rip: u64,
+        pub rdp: u64,
+        pub mxcsr: u32,
+        pub mxcsr_mask: u32,
+        pub st: [::upad128_t; 8],
+        pub xmm: [::upad128_t; 16],
+        pub __fx_ign: [::upad128_t; 6],
+        pub status: u32,
+        pub xstatus: u32,
+    }
+}
+
+s_no_extra_traits! {
+    #[cfg(libc_union)]
+    pub union __c_anonymous_fp_reg_set {
+        pub fpchip_state: __c_anonymous_fpchip_state,
+        pub f_fpregs: [[u32; 13]; 10],
```

Comment:
This really should be `[u32; 130]` but that may not work with older Rust versions used in CI.

@JohnTitor Thoughts?

Reply:
yes why I did just for precaution

Reply:
Yes, we've been using this workaround in some places and it should work fine.